geoengine/auth.py

```diff
@@ -37,9 +37,15 @@ class Session:
     __server_url: str
     __timeout: int = 60
 
+    __admin_token: Optional[UUID] = None
+
     session: ClassVar[Optional[Session]] = None
 
-    def __init__(self, server_url: str, credentials: Tuple[str, str] = None, token: str = None) -> None:
+    def __init__(self,
+                 server_url: str,
+                 credentials: Optional[Tuple[str, str]] = None,
+                 token: Optional[str] = None,
+                 admin_token: Optional[str] = None) -> None:
```

Comment:
what about loading from env?
add to the pydoc?

Reply:
[f2b4d39](https://github.com/geo-engine/geoengine-python/pull/83/commits/f2b4d398bf04c1f091a10d1550d9962b65a91035)